src/utils/github_updater.rs

```diff
@@ -109,27 +108,29 @@ fn get_github_fields(path: &str) -> Result<GitHubFields> {
                                         .and_then(|d| d.as_string())
                                         .unwrap_or(""),
                                     "%Y-%m-%dT%H:%M:%S")
-            .unwrap_or(time::now())
+            .unwrap_or_else(|_| time::now())
             .to_timespec(),
     })
 }
 
-
-
 fn get_github_path(url: &str) -> Option<String> {
     let re = Regex::new(r"https?://github\.com/([\w\._-]+)/([\w\._-]+)").unwrap();
     match re.captures(url) {
         Some(cap) => {
             let username = cap.get(1).unwrap().as_str();
             let reponame = cap.get(2).unwrap().as_str();
-            Some(format!("{}/{}",
-                         username,
-                         if reponame.ends_with(".git") {
-                             reponame.split(".git").nth(0).unwrap()
-                         } else {
-                             reponame
-                         }))
+
+            Some(format!(
+                "{}/{}",
+                username,
+                if reponame.ends_with(".git") {
+                   reponame.split(".git").next().unwrap()
+                } else {
+                    reponame
```

Comment:
Can you turn this into a temporary variable so it doesn't make this expression really big?